ts/src/exmo.ts

```diff
@@ -1329,34 +1329,43 @@ export default class exmo extends Exchange {
          * @name exmo#createOrder
          * @description create a trade order
          * @see https://documenter.getpostman.com/view/10287440/SzYXWKPi#80daa469-ec59-4d0a-b229-6a311d8dd1cd
+         * @see https://documenter.getpostman.com/view/10287440/SzYXWKPi#de6f4321-eeac-468c-87f7-c4ad7062e265  // stop market
+         * @see https://documenter.getpostman.com/view/10287440/SzYXWKPi#3561b86c-9ff1-436e-8e68-ac926b7eb523  // margin
          * @param {string} symbol unified symbol of the market to create an order in
          * @param {string} type 'market' or 'limit'
          * @param {string} side 'buy' or 'sell'
          * @param {float} amount how much of currency you want to trade in units of base currency
          * @param {float} [price] the price at which the order is to be fullfilled, in units of the quote currency, ignored in market orders
          * @param {object} [params] extra parameters specific to the exmo api endpoint
+         * @param {float} [params.stopPrice] the price at which a trigger order is triggered at
+         * @param {string} [params.timeInForce] *spot only* 'fok', 'ioc' or 'post_only'
+         * @param {boolean} [params.postOnly] *spot only* true for post only orders
          * @returns {object} an [order structure]{@link https://github.com/ccxt/ccxt/wiki/Manual#order-structure}
          */
         await this.loadMarkets ();
         const market = this.market (symbol);
-        const prefix = (type === 'market') ? (type + '_') : '';
-        const orderType = prefix + side;
         const isMarket = (type === 'market') && (price === undefined);
+        let marginMode = undefined;
+        [ marginMode, params ] = this.handleMarginModeAndParams ('createOrder', params);
+        const isSpot = (marginMode !== 'cross') && (marginMode !== 'isolated');
```

Comment:
isn't easier to do `isSpot = (marginMode === undefined)`

Reply:
I wanted to leave the option of setting `marginMode` equal to `spot` also

Reply:
@samgermain I think that is a bit weird and unnecessary